kdwsdl2cpp/common/fileprovider.cpp

```diff
@@ -53,6 +59,23 @@ void FileProvider::cleanUp()
   }
 }
 
+#if QT_VERSION >= QT_VERSION_CHECK(5, 4, 0)
+bool FileProvider::loadCertificate(const QString &certPath, const QString & password)
+{
+    QFile certFile(certPath);
+    if (certFile.open(QFile::ReadOnly)) {
```

Comment:
Would it make sense to report the error messages here? As this function knows what went wrong.